### PR TITLE
SatishMohanV's Bangalore addition

### LINKED ISSUE #20: Create a new file for SatishMohanV Bangalore, India

### PR DESCRIPTION
**Pull request by SatishMohanV**
close #20

### CODE REVIEW

**Comment by matthaldeman95** [comment]
👍

File: satishmohanv-bangalore.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Bangalore, India
+
```
Comment (matthaldeman95) [inline satishmohanv-bangalore.md]: Some members of my team live and work in Bangalore

Comment (SatishMohanV) [reply satishmohanv-bangalore.md]: 👍

**Review by matthaldeman95 — APPROVED** [review]
You've done a great job.  :1st_place_medal:

**Pull request merged**